src/components/DatePickerContainer/DatePickerContainer.css

```diff
@@ -0,0 +1,11 @@
+.infoPanel {
+	padding: 10px 21px;
+	border-top: 1px solid #dce0e0;
+	color: #484848;
+}
+
+.info {
```

Comment:
I think it was fine to remove it - we don't know what the content is yet or whether we need exclamation points.